presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -391,6 +392,19 @@ public FeaturesConfig setJoinReorderingStrategy(JoinReorderingStrategy joinReord
         return this;
     }
 
+    public int getMaxReorderedJoins()
```

Comment:
`@Min(2)`

Reply:
do we have a way to validate session properties?

Reply:
There is no API for the automatic validation. You can validate it maually though https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java#L727